db/migrate/20160203145841_add_content_id_to_about_page.rb

```diff
@@ -0,0 +1,16 @@
+require "securerandom"
+
+class AddContentIdToAboutPage < ActiveRecord::Migration
+  def up
+    add_column :about_pages, :content_id, :string
+
+    AboutPage.reset_column_information
+    AboutPage.all.each do |about_page|
+      about_page.update_attribute(:content_id, SecureRandom.uuid)
```

Comment:
Good point. It does seem sensible to me to treat separately the action of modifying the schema and of adding data. So this now only adds the content_id column, and the content_id generation is in a separate data migration, which is how it was done for classifications. @jamiecobbett any thoughts on us using this approach on future formats?